### PR TITLE
update shared subscription strategy

### PR DESCRIPTION
**Pull request by LenaLenaPan**

### CODE REVIEW

**Comment by qzhuyan** [comment]
we have 7 actually
   - `random`: dispatch the message to a random selected subscriber
   - `round_robin`: select the subscribers in a round-robin manner
   - `round_robin_per_group`: select the subscribers in round-robin fashion within each shared subscriber group
   - `local`: select random local subscriber otherwise select random cluster-wide
   - `sticky`: always use the last selected subscriber to dispatch, until the subscriber disconnects.
   - `hash_clientid`: select the subscribers by hashing the `clientIds`
   - `hash_topic`: select the subscribers by hashing the source topic"""
reference: https://github.com/emqx/emqx/pull/10079/files

File: en_US/mqtt/mqtt-shared-subscription.md [hunk 1]
```diff
@@ -56,12 +56,13 @@ broker.shared_subscription_strategy = random
...
-| Load Balance | Description                                 |
-| :----------  | :------------------------------------------ |
-| random       | Random selection among all subscribers      |
-| round_robin  | In order of subscription                    |
-| sticky       | Always send to the last selected subscriber |
-| hash         | Hash by publisher ClientID                  |
+| Load Balance  | Description                                        |
+| :------------ | :------------------------------------------------- |
+| random        | Random selection among all subscribers             |
+| round_robin   | In order of subscription                           |
```
Comment (LenaLenaPan) [inline en_US/mqtt/mqtt-shared-subscription.md]:
```suggestion
| `round_robin`   | Select the subscribers in a round-robin manner                     |
| `round_robin_per_group`   | Select the subscribers in round-robin fashion within each shared subscriber group                         |
| `local`   | Select a random local subscriber, if cannot be found, then select a random subscriber within the cluster                       |
```

File: en_US/mqtt/mqtt-shared-subscription.md [hunk 2]
```diff
@@ -56,12 +56,13 @@ broker.shared_subscription_strategy = random
...
+| sticky        | Always send to the last selected subscriber        |
```
Comment (LenaLenaPan) [inline en_US/mqtt/mqtt-shared-subscription.md]:
```suggestion
| sticky        | Always send to the last selected subscriber until the subscriber disconnects        |
```

File: en_US/mqtt/mqtt-shared-subscription.md [hunk 3]
```diff
@@ -56,12 +56,15 @@ broker.shared_subscription_strategy = random
...
 
-| Load Balance | Description                                 |
-| :----------  | :------------------------------------------ |
-| random       | Random selection among all subscribers      |
-| round_robin  | In order of subscription                    |
-| sticky       | Always send to the last selected subscriber |
-| hash         | Hash by publisher ClientID                  |
+| Load Balance  | Description                                        |
+| :------------ | :------------------------------------------------- |
+| random        | Random selection among all subscribers             |
```
Comment (LenaLenaPan) [inline en_US/mqtt/mqtt-shared-subscription.md]:
```suggestion
| `random`        | Random selection among all subscribers             |
```

File: en_US/mqtt/mqtt-shared-subscription.md [hunk 4]
```diff
@@ -56,12 +56,15 @@ broker.shared_subscription_strategy = random
...
+| `round_robin`   | Select the subscribers in a round-robin manner                     |
+| `round_robin_per_group`   | Select the subscribers in round-robin fashion within each shared subscriber group                         |
+| `local`   | Select a random local subscriber, if cannot be found, then select a random subscriber within the cluster                       |
+| sticky        | Always send to the last selected subscriber until the subscriber disconnects        |
```
Comment (LenaLenaPan) [inline en_US/mqtt/mqtt-shared-subscription.md]:
```suggestion
| `sticky`        | Always send to the last selected subscriber until the subscriber disconnects        |
```

File: en_US/mqtt/mqtt-shared-subscription.md [hunk 5]
```diff
@@ -56,12 +56,15 @@ broker.shared_subscription_strategy = random
...
+| hash_clientid | Select the subscribers by hashing the `clientIds`  |
```
Comment (LenaLenaPan) [inline en_US/mqtt/mqtt-shared-subscription.md]:
```suggestion
| `hash_clientid` | Select the subscribers by hashing the `clientIds`  |
```

File: en_US/mqtt/mqtt-shared-subscription.md [hunk 6]
```diff
@@ -56,12 +56,15 @@ broker.shared_subscription_strategy = random
...
+| hash_topic    | Select the subscribers by hashing the source topic |
```
Comment (LenaLenaPan) [inline en_US/mqtt/mqtt-shared-subscription.md]:
```suggestion
| `hash_topic`    | Select the subscribers by hashing the source topic |
```

File: zh_CN/mqtt/mqtt-shared-subscription.md [hunk 7]
```diff
@@ -60,7 +60,8 @@ broker.shared_dispatch_ack_enabled = false
 | random      | 在所有订阅者中随机选择       |
```
Comment (LenaLenaPan) [inline zh_CN/mqtt/mqtt-shared-subscription.md]:
```suggestion
| `random`      | 在所有订阅者中随机选择       |
```

File: zh_CN/mqtt/mqtt-shared-subscription.md [hunk 8]
```diff
@@ -60,7 +60,8 @@ broker.shared_dispatch_ack_enabled = false
...
 | round_robin | 按照订阅顺序                 |
```
Comment (LenaLenaPan) [inline zh_CN/mqtt/mqtt-shared-subscription.md]:
```suggestion
| `round_robin` | 按照订阅顺序选择                 |
| `round_robin_per_group` | 在每个共享订阅组中按照订阅顺序进行选择                 |
| `local ` | 随机在本地订阅中进行选择，如无法找到，则在集群范围内随机选择                 |
```

File: zh_CN/mqtt/mqtt-shared-subscription.md [hunk 9]
```diff
@@ -60,7 +60,8 @@ broker.shared_dispatch_ack_enabled = false
...
 | sticky      | 一直发往上次选取的订阅者     |
```
Comment (LenaLenaPan) [inline zh_CN/mqtt/mqtt-shared-subscription.md]:
```suggestion
| `sticky`      | 选定订阅者后，始终向其进行发送，直到该订阅者断开连接   |
```

File: zh_CN/mqtt/mqtt-shared-subscription.md [hunk 10]
```diff
@@ -60,7 +60,8 @@ broker.shared_dispatch_ack_enabled = false
...
-| hash        | 按照发布者 ClientID 的哈希值 |
+| hash_clientid | 通过对发送者的客户端 ID 进行 Hash 处理来选择订阅者 |
```
Comment (LenaLenaPan) [inline zh_CN/mqtt/mqtt-shared-subscription.md]:
```suggestion
| `hash_clientid` | 通过对发送者的客户端 ID 进行 Hash 处理来选择订阅者 |
```

File: zh_CN/mqtt/mqtt-shared-subscription.md [hunk 11]
```diff
@@ -60,7 +60,8 @@ broker.shared_dispatch_ack_enabled = false
...
+| hash_topic | 通过对源主题进行 Hash 处理来选择订阅者 |
```
Comment (LenaLenaPan) [inline zh_CN/mqtt/mqtt-shared-subscription.md]:
```suggestion
| `hash_topic` | 通过对源主题进行 Hash 处理来选择订阅者 |
```

**Comment by LenaLenaPan** [comment]
@qzhuyan updated as suggested, pls have another look.

**Review by wivwiv — APPROVED** [review]

**Pull request merged**